prow/plugins/releasenote/releasenote.go

```diff
@@ -45,12 +45,15 @@ const (
 	releaseNoteFormat       = `Adding the "%s" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.`
 	parentReleaseNoteFormat = `All 'parent' PRs of a cherry-pick PR must have one of the %q or %q labels, or this PR must follow the standard/parent release note labeling requirement.`
 
-	actionRequiredNote = "action required"
+	actionRequiredNote           = "action required"
+	deprecationLabel             = "kind/deprecation"
```

Comment:
Can you please group this label with the ones on L41-43?

Reply:
Thank you for the review. Fixed.